### PR TITLE
github-user

### PR DESCRIPTION
**Pull request by maccrin**
Github-Users-Search-PR

### CODE REVIEW

File: react/react2/react2-week1/github-user/src/components/Search.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const usingContext = useContext(userContext);
+
+  return (
+    <div className="container">
+      <input
+        type="text"
+        placeholder="Enter the User name"
+        onChange={usingContext.search}
+      />
+      <>
```
Comment (Divyajg) [inline react/react2/react2-week1/github-user/src/components/Search.jsx:15]: Using react fragments is not a good practice, instead you can use `<div>`

**Review by Divyajg — APPROVED** [review]
Very good job.
One more idea to pass properties. You can also pass `useState variables like input, setInput` and you can use `onChange={usingContext.setInput}`. This is one of the great advantages with Context-concept.

File: react/react2/react2-week1/github-user/src/components/Search.jsx (hunk 1 above)
Comment (maccrin) [reply react/react2/react2-week1/github-user/src/components/Search.jsx:15]: 🙂 sure will follow and thank you very much for your time 🙏🏾.

**Comment by maccrin** [comment]
> Very good job. One more idea to pass properties. You can also pass `useState variables like input, setInput` and you can use `onChange={usingContext.setInput}`. This is one of the great advantages with Context-concept.

Yes,when I did this homework 3 weeks back 1st.time learned context so was not .much comfortable but now I tried to learn and still trying apply as you recommended in my current code 🙏🏾.
Thank you very much for review 🙂.

**Pull request merged**